docs/samples/Microsoft.ML.Samples/Dynamic/ValueMappingStringToArray.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Conversions;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ValueMappingStringToArrayExample
```

Comment:
>ValueMappingStringToArrayExample [](start = 17, length = 32)

this is not referenced anywhere.

Reply:
RIght -  this is just another example that is available to show how this can be used. There is another example that is not used either. Does they all need to be referenced?

---
In reply to: [252891445](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252891445) [](ancestors = 252891445)

Reply:
I think we should remove them. The purpose of this project is to populate the API reference website. 

the machinelearning-samples repo is the one where we can put more samples, if needed. 

If we keep them here, unreferenced, it will be hard to see what is used for what, overtime. 

---
In reply to: [252938208](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252938208) [](ancestors = 252938208,252891445)

Reply:
Is it possible to add multiple links? I initially had these in one file but moved out into separate files to not have to update line number reference.

---
In reply to: [252938844](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252938844) [](ancestors = 252938844,252938208,252891445)

Reply:
I added multiple links so they are all referenced now.

---
In reply to: [252939988](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252939988) [](ancestors = 252939988,252938844,252938208,252891445)